tests/foreman/api/test_errata.py

```diff
@@ -229,7 +233,7 @@ def test_positive_install_multiple_in_host(
             search_rate=20,
             max_tries=15,
         )
-        sleep(10)
+        sleep(20)
```

Comment:
why sleep time?

Reply:
failed prt see previous pr, needs this delay.
https://github.com/SatelliteQE/robottelo/pull/12706

Reply:
There is an asynchronous task to wait for after errata install, to update the list of applicable errata